openfecwebapp/templates/macros/filters/election-filter.html

```diff
@@ -25,11 +25,13 @@
       <option value="{{ year }}">{{ year }}</option>
     {% endfor %}
   </select>
+  {% if office in ['president', 'senate'] %}
```

Comment:
Nope. We don't show the time period for house races (because they're only two years).